apollo-federation/src/link/cost_spec_definition.rs

```diff
@@ -147,11 +147,6 @@ impl CostSpecDefinition {
         apollo_compiler::ast::DirectiveList,
         Node::new
     );
-    propagate_demand_control_directives!(
```

Comment:
@tninesling The `propagate_demand_control_schema_directives` function generated by this macro is dead code. Is it meant to be used?

Reply:
That's fine to be deleted. Looks like I forgot to delete it when I replaced it with the `to_position` version underneath